pymoo/core/problem.py

```diff
@@ -57,6 +57,21 @@ def __getstate__(self):
         state.pop("client", None)
         return state
 
+class JoblibParallelization:
+
+    def __init__(self, parallel, delayed, *args, **kwargs) -> None:
```

Comment:
in favour of clean code, i would have preferred somthing similar to ...
```suggestion
    def __init__(self, aJoblibParallel, aJoblibDelayed, *args, **kwargs) -> None:
```
... rendering the comments obsolete